CONTRIBUTING.md

```
@@ -189,8 +189,57 @@
 - **release branch**: Merging from `stage` to a `release branch` prepares your code for deployment to **Production**.
 - **main**: Once your release branch is merged into `main`, your code is officially live in the **Production** environment.
 
-**Summary:**  
-Each branch merge advances your code to the next environment in the pipeline.
+#### Step-By-Step Migration
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)